### PR TITLE
Suggestions from R9 analyzers - round 1

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: src/Microsoft.Identity.Web.TokenAcquisition/AspNetCore/TokenAcquisition-AspnetCore.cs [hunk 1]
```diff
@@ -142,17 +140,9 @@ internal class TokenAcquisitionAspNetCore : TokenAcquisition, ITokenAcquisition,
         public async Task AddAccountToCacheFromAuthorizationCodeAsync(
             AuthorizationCodeReceivedContext context,
             IEnumerable<string> scopes,
-            string authenticationScheme /*= OpenIdConnectDefaults.AuthenticationScheme*/)
+            string authenticationScheme = OpenIdConnectDefaults.AuthenticationScheme /*= OpenIdConnectDefaults.AuthenticationScheme*/)
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web.TokenAcquisition/AspNetCore/TokenAcquisition-AspnetCore.cs:143]: I'm not sure about this one. Let's just take a note shouldn't prevent you from merging

File: src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirerFactory-GetTokenAcquirers.cs [hunk 2]
```diff
@@ -112,6 +107,14 @@ public ITokenAcquirer GetTokenAcquirer(string authenticationScheme = "")
             }
             return acquirer;
         }
+        private void CheckServiceProviderNotNull()
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirerFactory-GetTokenAcquirers.cs:110]: I wonder about the performance impact. Is it an r9 rule?

Comment (jennyf19) [reply src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirerFactory-GetTokenAcquirers.cs:110]: They do something else for exceptions around parameter checks, which I will do next.

File: src/Microsoft.Identity.Web.TokenAcquisition/AspNetCore/TokenAcquisition-AspnetCore.cs (hunk 1 above)
Comment (jennyf19) [reply src/Microsoft.Identity.Web.TokenAcquisition/AspNetCore/TokenAcquisition-AspnetCore.cs:143]: The interface defines the default.

**Pull request merged**